idaes/core/dmf/tests/test_getver.py

```diff
@@ -73,12 +73,9 @@ class fake_loader:
 
 @pytest.mark.unit
 def test_more_func():
-    import six
-
-    # doesn't have __path__ but has __loader__ with path
-    dmf.get_version_info(six)
```

Comment:
From what I can understand from the context, `six` is being used here as an example of a module that has an empty `__path__` attribute, which in turn I imagine is used to test whether the DMF is able to deal with such cases. Pinging @dangunter in case he wants to chime in on this.

Reply:
That was my interpretation too, but it runs into the issue of either:

a) `six` is not a direct dependency of IDAES so it might not be there, or
b) adding `six` as a dependency just to test this functionality.

Given the lack of use of the DMF, I took the easy route and just removed this part of the test. Also, I will note this test does not assert anything (and I almost deleted the whole test for that reason).

Reply:
I agree that taking the test out seems the most reasonable thing as having `six` as a dependency just for this test seems unnecessary. Specifically about the lack of `assert` - I think it's just "asserting it doesn't fail", i.e. that calling the function doesn't cause an exception to be raised.

Reply:
Right. At one point six was a direct dependency of IDAES, but of course I didn't remember that this test existed when six was removed (probably by me, but ¯\_(ツ)_/¯ ). Removing the test is fine. The DMF is at the back of the refrigerator right now waiting for the next cleaning.